docs/Makefile

```diff
@@ -31,7 +31,8 @@ report_queries:
 # Convert each SPARQL file to a Markdown file.
 report_queries/%.md: ../robot-core/src/main/resources/report_queries/%.rq | report_queries
 	awk '/^PREFIX/ && !x {print "```sparql"; x=1} 1' $< \
-	| sed 's/^# \? \?//' \
+	| sed 's/^#//' \
+	| sed 's/^ //' \
```

Comment:
FYI I changed this bit of the `docs/Makefile` on another branch to make the indentation better, which changes a all the `docs/report_queries/*.md` files.